modules/javafx.web/src/test/java/test/javafx/scene/web/LocalStorageTest.java

```diff
@@ -80,6 +85,7 @@ public void testLocalStorage() throws Exception {
         webEngine.setJavaScriptEnabled(true);
         webEngine.setUserDataDirectory(LOCAL_STORAGE_DIR);
         checkLocalStorageAfterWindowClose(webEngine);
+        webEngine.setUserDataDirectory(null);
```

Comment:
Since this cleanup is done as the last statement of each test, maybe it would be better to move this call to an `@After` method (note: _not_ `@AfterClass`)?